icdc_schema.py

```diff
@@ -37,8 +37,8 @@
 DESCRIPTION = 'Desc'
 
 
-def get_list_values(list_str):
-    return [item.strip() for item in list_str.split(LIST_DELIMITER) if item.strip()]
+def get_list_values(list_str, list_delimiter):
```

Comment:
I suggest moving **get_list_values** function inside ICDC_Schema class, so that it can use the delimiter in the class/object, instead of asking caller to provide delimiter.